packages/forms/src/UIForm/fieldsets/Fieldset/Fieldset.component.test.js

```diff
@@ -29,6 +29,44 @@ describe('Fieldset widget', () => {
 		expect(wrapper.getElement()).toMatchSnapshot();
 	});
 
+	it('should not render fieldset with empty items', () => {
+		// given
+		const schema = {
+			title: 'My fieldset',
+			items: [
+				{
+					key: ['user', 'firstname'],
+					type: 'text',
+					schema: { type: 'string' },
+				},
+				{
+					key: ['user', 'lastname'],
+					type: 'text',
+					schema: { type: 'string' },
+				},
+			],
+		};
+
+		// when
+		const wrapper = shallow(<Fieldset schema={schema} />);
+
+		// then
+		expect(wrapper.getElement()).toMatchSnapshot();
+	});
+
+	it('should not render fieldset legend without any title', () => {
+		// given
+		const schema = {
```

Comment:
Maybe this test need items, to focus of shallow a render with fieldset, but no title?